gse_Conversion/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+
+{
+    "name": "Gse conversion USD ",
+    "summary": """
+        Customisation of Gse conversion  module for GoShop Energy""",
```

Comment:
@MaganaAsiati 
it's not a customisation of the module conversion, you create a new module for the conversion...